### PR TITLE
feat-post

### PR DESCRIPTION
**Pull request by deepak29-git**
1. create post 
2. edit post
3. delete post
4. like post
5. dislike post

image

### CODE REVIEW

**Review by Mohitkumar33 — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -26,7 +28,7 @@
     -->
     <title>React App</title>
   </head>
-  <body>
+  <body style="background-color:#f3f2ef ;">
```
Comment (shivanigangadharan) [inline public/index.html:31]: Preferably avoid inline styling on body tag, can give class.

File: src/Components/Sidebar/Sidebar.css [hunk 2]
```diff
@@ -7,8 +7,12 @@
 }
 
 .list-container {
-  border-right: 1px solid var(--grey-color);
-  width: 250px;
+  background-color: var(--white-color);
   position: fixed;
+  width: 280px;
```
Comment (shivanigangadharan) [inline src/Components/Sidebar/Sidebar.css:12]: Can keep this in rem instead of px

File: src/Custom-hook/user-api.js [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+//         const { data } = await axios.get("/api/users");
+//         setLoader(false);
+//         setData(data.users);
+//       } catch (error) {
+//         console.log(error);
+//       }
+//     })();
+//   }, []);
+//   return { data, loader };
+// };
```
Comment (shivanigangadharan) [inline src/Custom-hook/user-api.js:21]: Should remove unwanted comments in this file

File: src/backend/db/posts.js [hunk 4]
```diff
@@ -7,30 +7,30 @@ import { formatDate } from "../utils/authUtils";
...
+  //     "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio. Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus. Temporibus autem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut et voluptates repudiandae sint et molestiae non recusandae. Itaque earum rerum hic tenetur a sapiente delectus, ut aut reiciendis voluptatibus maiores alias consequatur aut perferendis doloribus asperiores repellat.",
+  //   likes: {
+  //     likeCount: 0,
+  //     likedBy: [],
+  //     dislikedBy: [],
+  //   },
+  //   username: "shubhamsoni",
+  //   createdAt: formatDate(),
+  //   updatedAt: formatDate(),
+  // },
```
Comment (shivanigangadharan) [inline src/backend/db/posts.js:35]: Can remove this too

**Review by shivanigangadharan — APPROVED** [review]
Left some minor suggestions, rest of the code looks good. I like how you've structured function for each operation. Nice UI as well. 👍🏼

File: public/index.html (hunk 1 above)
Comment (deepak29-git) [reply public/index.html:31]: ok

File: src/Custom-hook/user-api.js (hunk 3 above)
Comment (deepak29-git) [reply src/Custom-hook/user-api.js:21]: ok

File: src/backend/db/posts.js (hunk 4 above)
Comment (deepak29-git) [reply src/backend/db/posts.js:35]: ok

**Pull request merged**